openasip/src/applibs/ProGe/TestBenchBlock.cc

```diff
@@ -102,15 +102,11 @@ TestBenchBlock::~TestBenchBlock() {
 void
 TestBenchBlock::write(const Path& targetBaseDir, HDL targetLang) const {
     // Check language compatibility //
-    std::cout << "--1" << std::endl;
```

Comment:
Can you squash this with the commit that introduced the debug prints. Now there's 77 commits which is quite a lot? Fixup commits help to clean up the history.